app/app/urls.py

```diff
@@ -113,6 +113,9 @@
     ),
     re_path(r'^funding/details/?', dashboard.views.bounty_details, name='funding_details'),
 
+    url(r'^issue/?', dashboard.views.bounty_details, name='bounty_details'),
```

Comment:
whats with the dupe URLS here?

Reply:
These were originally removed by https://github.com/gitcoinco/web/pull/1887, which prevented non-github bounties from rending. I then committed these changes. Following this, another core member merged a4e146, which is responsible for the duplication you're seeing here. 

I've been trying to keep up with the changes & bugs being introduced into master, but missed the resolution these changes were intended to fix.